.eslintrc.js

```diff
@@ -29,52 +23,34 @@ module.exports = {
   },
   overrides: [
     {
-      // ES6/browser processed by Babel
+      // NodeJS build tools
       files: [
-        'src/**/*',
-        'test/jest/**/*',
-        'test/e2e/harness/src/**/*',
-        'test/e2e/harness/e2e/**/*',
-        'rollup.config.js'
+        'build.js', 
+        'env.js', 
+        'util/**/*', 
+        'config-overrides.js',
+        'protractor.conf.js'
+      ],
+      extends: [
+        'eslint:recommended',
```

Comment:
I'm not sure that "extends" works on an overrides (I think it doesn't). You can add plugins though.

Reply:
probably that's why it works, https://github.com/eslint/eslint/issues/8813#issuecomment-320459067

Switched to plugins.